examples/Folders/Readme.md

```diff
@@ -0,0 +1,49 @@
+Dropzone that accepts folders as drag-and-drop.
+
+```
+const {getDroppedOrSelectedFiles} = require('html5-file-selector')
+
+class FolderDropzone extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: [] }
+  }
+
+  onDrop(files) {
+    this.setState({
+      files
+    })
+  }
+
+  render() {
+    return (
+      <section>
+        <div className="dropzone">
+          <Dropzone
+            getDataTransferItems={evt =>
+              getDroppedOrSelectedFiles(evt).then(list =>
+                list.map(({ fileObject, ...rest }) => fileObject)
+              )
+            }
+            onDrop={this.onDrop.bind(this)}
+          >
+            <p>Drop a folder with files here.</p>
+          </Dropzone>
+        </div>
+        <aside>
+          <h2>Dropped files</h2>
+          <ul>
+            {this.state.files.map(f => (
+              <li key={f.name}>
+                {f.name} - {f.size} bytes
```

Comment:
We should display the folder structure here so the example is clear about what it implements. Does the plugin we're using supports that? If not, we should at least show full paths here.

Reply:
I couldn't find anything to display folder structure in the plug-in, so I will add full paths of dropped files for now so that it's clear what plug-in is responsible for.